enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -61,22 +86,27 @@ type MetricsCollector struct {
 	database    database.Store
 	logger      slog.Logger
 	snapshotter prebuilds.StateSnapshotter
+
+	replacementsCounter   map[replacementKey]*atomic.Int64
+	replacementsCounterMu sync.Mutex
 }
 
 var _ prometheus.Collector = new(MetricsCollector)
 
 func NewMetricsCollector(db database.Store, logger slog.Logger, snapshotter prebuilds.StateSnapshotter) *MetricsCollector {
 	return &MetricsCollector{
-		database:    db,
-		logger:      logger.Named("prebuilds_metrics_collector"),
-		snapshotter: snapshotter,
+		database:            db,
+		logger:              logger.Named("prebuilds_metrics_collector"),
+		snapshotter:         snapshotter,
+		replacementsCounter: make(map[replacementKey]*atomic.Int64),
```

Comment:
Nit: Since this is always accessed within mutex lock, consider using plain `int64`. Simplifies the code as well as we don't need to initialize the pointer to atomic int.

(Didn't look too closely at this last time around to spot the discrepancy.)